design-doc-template.adoc

```diff
@@ -0,0 +1,39 @@
+= Title
+:author:            Your Name
+:email:             your.email@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
```

Comment:
This is wrong to begin with, the correct url prefix is `https://issues.jboss.org/browse/`

Though, this really goes against the ideas behind asciidoc which is to be human readable even unrendered. Let alone the fact that it now generates wrong links..